contents/docs/csp-tracking/index.mdx

```diff
@@ -7,10 +7,10 @@ showTitle: true
 You can add a Content Security Policy (CSP) to your website to prevent cross-site scripting (XSS) attacks.
 This is a security feature built into modern browsers that helps protect your users' data and privacy.
 
-You can send reports of CSP rule violations to PostHog, which is useful for
-* warning when your website is under certain kinds of attack
-* debugging problems when adding external scripts/media/etc to your site
-* being confident that changes to your site haven't broken the loading of any resources
+You can [send reports of CSP rule violations](/docs/csp-tracking) to PostHog, which is useful for:
```

Comment:
```suggestion
You can send reports of CSP rule violations to PostHog, which is useful for:
```

Reply:
lol o wait is the same page :P

Reply:
i came here to say this lol, linking to itself